diff --git a/node_modules/pinia/dist/pinia.mjs b/node_modules/pinia/dist/pinia.mjs
index b0095a4..0db0a6d 100644
--- a/node_modules/pinia/dist/pinia.mjs
+++ b/node_modules/pinia/dist/pinia.mjs
@@ -2002,4 +2002,4 @@ const PiniaVuePlugin = function (_Vue) {
     });
 };
 
-export { MutationType, PiniaVuePlugin, acceptHMRUpdate, createPinia, defineStore, getActivePinia, mapActions, mapGetters, mapState, mapStores, mapWritableState, setActivePinia, setMapStoreSuffix, skipHydrate, storeToRefs };
+export { MutationType, PiniaVuePlugin, acceptHMRUpdate, createPinia, defineStore, getActivePinia, mapActions, mapGetters, mapState, mapStores, mapWritableState, setActivePinia, setMapStoreSuffix, skipHydrate, storeToRefs, registerPiniaDevtools };
